src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -24,12 +26,64 @@ internal static class Globals
         internal const string LogTag = "Tizen.Sensor";
     }
 
+    /// <summary>
+    /// Data from sensor
+    /// </summary>
+    public class SensorEvent
```

Comment:
Why not `SensorData`?
This class summary is `Data from sensor`. So, this class named as SensorData not Event